components/cards/WhatWeDoCard.jsx

```diff
@@ -0,0 +1,42 @@
+import { motion, useAnimation } from "framer-motion";
+import { useInView } from "react-intersection-observer";
+import { useEffect } from "react";
+import Image from 'next/image'
+import { H4 } from '../ui/Headings';
+
+const WhatWeDo = ({index, title, description, image}) => {
+    const boxVariant = {
+        visible: { opacity: 1, scale: 1, x: 0 , transition: {duration: 0.5} },
+        hidden: { opacity: 0, scale: 0, x: index % 2 == 0 ? 100 : -100 },
+    }
+    
+    const control = useAnimation()
+    const [ref, inView] = useInView()
+
+    useEffect(() => {
+        if (inView) {
+          control.start("visible");
+        } else {
+            control.start("hidden");
```

Comment:
As this is only an `if else` statement with one value handled, you just refactor the code using ternary operator.

Reply:
Applied